client/component/comment.js

```diff
@@ -273,6 +275,14 @@ export function EditComment({comment, big=false, setComment, topLevel, onEditing
                         <HorizBox center right>
                             {onCancel && <PadBox right={20}><TextButton color={colorTextGrey} label='Cancel' onPress={onCancel} /></PadBox>}
                             <CTAButton label={action} disabled={!canPost || inProgress} type='primary' onPress={onPost} />
+                            <PadBox left={20}>
```

Comment:
Should be in _commentEditBottomWidgets_ to make it more agnostic of the feature?